Jenkinsfile

```diff
@@ -0,0 +1,576 @@
+        '1.21':    'kindest/node:v1.21.1@sha256:69860bda5563ac81e3c0057d654b5253219618a22ec3a346306239bba8cfa1a6',
+        '1.20.7':  'kindest/node:v1.20.7@sha256:cbeaf907fc78ac97ce7b625e4bf0de16e3ea725daf6b04f930bd14c67c671ff9',
+        '1.20':    'kindest/node:v1.20.7@sha256:cbeaf907fc78ac97ce7b625e4bf0de16e3ea725daf6b04f930bd14c67c671ff9',
+        '1.19.11': 'kindest/node:v1.19.11@sha256:07db187ae84b4b7de440a73886f008cf903fcf5764ba8106a9fd5243d6f32729',
+        '1.19':    'kindest/node:v1.19.11@sha256:07db187ae84b4b7de440a73886f008cf903fcf5764ba8106a9fd5243d6f32729',
+        '1.18.19': 'kindest/node:v1.18.19@sha256:7af1492e19b3192a79f606e43c35fb741e520d195f96399284515f077b3b622c',
+        '1.18':    'kindest/node:v1.18.19@sha256:7af1492e19b3192a79f606e43c35fb741e520d195f96399284515f077b3b622c',
+        '1.17.17': 'kindest/node:v1.17.17@sha256:66f1d0d91a88b8a001811e2f1054af60eef3b669a9a74f9b6db871f2f1eeed00',
+        '1.17':    'kindest/node:v1.17.17@sha256:66f1d0d91a88b8a001811e2f1054af60eef3b669a9a74f9b6db871f2f1eeed00',
+        '1.16.15': 'kindest/node:v1.16.15@sha256:83067ed51bf2a3395b24687094e283a7c7c865ccc12a8b1d7aa673ba0c5e8861',
+        '1.16':    'kindest/node:v1.16.15@sha256:83067ed51bf2a3395b24687094e283a7c7c865ccc12a8b1d7aa673ba0c5e8861',
+        '1.15.12': 'kindest/node:v1.15.12@sha256:b920920e1eda689d9936dfcf7332701e80be12566999152626b2c9d730397a95',
+        '1.15':    'kindest/node:v1.15.12@sha256:b920920e1eda689d9936dfcf7332701e80be12566999152626b2c9d730397a95'
+    ]
+]
+def _kind_image = ''
+
+pipeline {
+    agent { label 'VM.Standard2.8' }
+    options {
+//        skipDefaultCheckout(true)
+        timeout(time: 800, unit: 'MINUTES')
+    }
+
+    tools {
+        maven 'maven-3.8.4'
+        jdk 'OpenJDK 15.0.2'
+    }
+
+    environment {
+        github_url = "${env.GIT_URL}"
+        github_creds = "ecnj_github"
+        dockerhub_username_creds = "docker-username"
+        dockerhub_password_creds = "docker-password"
+        ocr_username_creds = 'OCR username'
+        ocr_password_creds = 'OCR Password'
+        ocir_registry_creds = 'ocir-server'
+        ocir_email_creds = 'ocir-email'
+        ocir_username_creds = 'ocir-username'
+        ocir_password_creds = 'ocir-token'
+        image_pull_secret_weblogic_creds = 'image-pull-secret-weblogic'
+
+        outdir = "${WORKSPACE}/staging"
+        result_root = "${outdir}/wl_k8s_test_results"
+        pv_root = "${outdir}/k8s-pvroot"
+        kubeconfig_file = "${result_root}/kubeconfig"
+
+        kind_name = "kind"
+        kind_network = "kind"
+        registry_name = "kind-registry"
+        registry_host = "${registry_name}"
+        registry_port = "5000"
+
+        start_time = sh(script: 'date +"%Y-%m-%d %H:%M:%S"', returnStdout: true).trim()
+        wle_download_url="https://github.com/oracle/weblogic-logging-exporter/releases/latest"
+        wko_build_email='robert.patrick@oracle.com'
```

Comment:
Can we use the ecnj_sonar account? This seems like the kind of thing we don't want CorpArch to find if they do an audit.

Reply:
Of course.  As it turns out this Jenkins server's email configuration is still pointing at stbeehive so email doesn't work.  Unless we fix this, we don't need at email address anyway.  One downside of using this generic account's email address is that someone will have to be checking that user's email to see failure emails.  Might be better to use a mailing list instead.

Reply:
Derek had me sign-up for that account, so I get the reports. Now that I think about it, the ecnj_sonar account doesn't make any sense for this other than that it isn't a personal account. I'm really okay not getting or sending e-mail about build failures.